dask/dataframe/io/parquet/arrow.py

```diff
@@ -762,6 +768,30 @@ def write_metadata(cls, parts, meta, fs, path, append=False, **kwargs):
     # Private Class Methods
     #
 
+    @classmethod
+    def _default_open_file_options(cls, fs, storage_options, input_options):
```

Comment:
Sidenote: I agree with Matt that it'd be nice to inline this logic, since it's relatively straightward and only used in one place. However, my guess is inlining this logic will make it more difficult to test. If that's the case, then I think testability should take priority.